packages/analytics-js/src/components/utilities/url.ts

```diff
@@ -6,7 +6,7 @@ import { Nullable } from '@rudderstack/analytics-js/types';
  * @returns url
  */
 const removeTrailingSlashes = (url: string | null): Nullable<string> =>
-  url && url.endsWith('/') ? url.replace(/\/+$/, '') : url;
+  url && url.endsWith('/') ? url.substring(0, url.length - 1) : url;
```

Comment:
There could be more than one trailing slash. I know the previous regex resulted in a bug but try to find something different.

Reply:
Done